package.json

```diff
@@ -62,9 +66,13 @@
     "nyc": "^15.1.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-modal": "^3.16.1",
     "start-server-and-test": "^2.0.0",
     "styled-components": "^6.0.0-rc.3",
     "supabase": "^1.68.6",
     "typescript": "5.0.4"
+  },
+  "devDependencies": {
+    "@types/react-modal": "^3.16.0"
```

Comment:
can this be all in dependencies (rather than dev)? we're not splitting dependencies for the time being :)

Reply:
I have moved it